kubernetes/cray-ims/templates/cray-ims-v2-image-create-kiwi-ng-job-template.yaml

```diff
@@ -197,7 +209,7 @@ data:
                 cpu: "500m"
               limits:
                 memory: "$job_mem_limit"
-                cpu: "48"
+                cpu: "8"
```

Comment:
When IMS was first being implemented, builds took a long time (over 30 minutes is my recollection), but when we bumped up the cpu limits they ran much faster (5 minutes is my recollection), which is why the cpu limit was made so large.  A lot has changed since then, but are build times still reasonable with dropping the limit down to only 8 cpus?

Reply:
Okay, I just saw the change for the kata containers to bump them up to 16 vcpu, so why is the cpu limit here half of that?

Reply:
To give it some overhead and allow for multiple containers that may be hitting it up. This is also adjustable much more readily than the kata config, so if the users decide to bump it up it has some extra room.

Reply:
We were having problems with the kata VM getting overwhelmed and killed out from under us.

Reply:
Okay, thanks for the explanation